test/development/acceptance-app/rsc-runtime-errors.test.ts

```diff
@@ -50,83 +52,37 @@ describe('Error overlay - RSC runtime errors', () => {
 
     const browser = await next.browser('/client')
 
-    await assertHasRedbox(browser)
-
-    const errorDescription = await getRedboxDescription(browser)
-    expect(errorDescription).toContain(
-      'Error: `cookies` was called outside a request scope. Read more: https://nextjs.org/docs/messages/next-dynamic-api-wrong-context'
-    )
-  })
-
-  it('should show source code for jsx errors from server component', async () => {
-    await next.patchFile(
-      'app/server/page.js',
-      outdent`
-        export default function Page() {
-          return <div>{alert('warn')}</div>
-        }
-      `
-    )
-
-    const browser = await next.browser('/server')
-    await assertHasRedbox(browser)
-
-    const errorDescription = await getRedboxDescription(browser)
-
-    expect(errorDescription).toContain(`Error: alert is not defined`)
-  })
-
-  it('should show the userland code error trace when fetch failed error occurred', async () => {
-    await next.patchFile(
-      'app/server/page.js',
-      outdent`
-        export default async function Page() {
-          await fetch('http://locahost:3000/xxxx')
-          return 'page'
-        }
-        `
-    )
-    const browser = await next.browser('/server')
-    await assertHasRedbox(browser)
-
-    const source = await getRedboxSource(browser)
-    // Can show the original source code
-    expect(source).toContain('app/server/page.js')
-    expect(source).toContain(`await fetch('http://locahost:3000/xxxx')`)
-  })
-
-  it('should contain nextjs version check in error overlay', async () => {
```

Comment:
Tested in https://github.com/vercel/next.js/blob/cbd1f7c30d0e601b7924f40fb18a78c25fd55d34/test/development/acceptance-app/version-staleness.test.ts